### DISCUSSION TITLE
how to compile windterm?

### DISCUSSION THREAD

**Discussion by buzhaodi** [kingToolbox/WindTerm#378] [Q&A]
i test compile for windterm on windoows 11,but it have some troubule,Is there a tutorial?
这个是要用什么去构建啊？

**Comment by mlsqq** [comment]
估计是查看不了，每次重开软件都要输入密码

**Comment by kingToolbox** [comment]
@buzhaodi 
Sorry for the late reply, `WindTerm` is a partially open source project and therefore cannot be compiled at this time.
